gateway/gateway_test.go

```diff
@@ -311,6 +327,8 @@ func TestSkipTargetPassEscapingOffWithSkipURLCleaningTrue(t *testing.T) {
 			spec.Proxy.StripListenPath = true
 			spec.Proxy.ListenPath = "/listen_me"
 			spec.Proxy.TargetURL = TestHttpAny + "/sent_to_me"
+			// change name to change checksum and reload
+			spec.Name = t.Name()
```

Comment:
I wonder if changing "Name" (adding some random prefix), should be default behaviour of `BuildAndLoadAPI` so when you write a test you do not have to think about it. And IF you actually writing test to check API caching functionality, you will actually force spec.Name be same between reloads.